app/resources/css-default.template

```diff
@@ -628,7 +628,7 @@ img.icon {
 
 .view input.add-note {
   width: 100%;
-  margin-top: 10px;
+  margin-top: 10px; /* Vertically align blue buttom with record card */
```

Comment:
Should be: "To vertically align the "Add note" button with the record card"

For spell correction and more clarification.